### PR TITLE
added dana inquiry user info function

### PR DESCRIPTION
**Pull request by mellyzarsmwn**
## What does this PR do?
Added dana inquiry user info function

## Why are we doing this? Any context or related work?
[[BE - Sangu] Correct DANA endpoint to get users' detail](https://kitabisa.atlassian.net/browse/UPS-3845)

## Where should a reviewer start?
Not much, please see changed file.

## Database changes
_optional -- If there's database changes, put it here_

## Config changes
_optional -- If there's config changes, put it here_

## Deployment instructions
_optional -- Better to put it if there's some 'special case' for deployment_

### CODE REVIEW

File: core.go [hunk 1]
```diff
@@ -23,12 +23,14 @@ const (
...
 	DANA_TIME_LAYOUT        = "2006-01-02T15:04:05-07:00"
 	CURRENCY_IDR            = "IDR"
+	INQUIRY_USER_INFO_PATH  = "v1/customers/user/inquiryUserInfoByAccessToken.htm"
 
 	FUNCTION_CREATE_ORDER       = "dana.acquiring.order.createOrder"
 	FUNCTION_QUERY_ORDER        = "dana.acquiring.order.query"
 	FUNCTION_REFUND             = "dana.acquiring.refund.refund"
 	FUNCTION_APPLY_ACCESS_TOKEN = "dana.oauth.auth.applyToken"
 	FUNCTION_USER_PROFILE       = "dana.member.query.queryUserProfile"
+	FUNCTION_INQUIRY_USER_INFO  = "customers.openapi.user.inquiryUserInfoByAccessToken"
```
Comment (sswastioyono18) [inline core.go:33]: ini tu digunain untuk apa yak mel?

File: core.go [hunk 2]
```diff
@@ -211,3 +234,61 @@ func (gateway *CoreGateway) requestToDana(reqBody interface{}, accessToken strin
 
 	return
 }
+
+func (gateway *CoreGateway) requestToDanaV1(reqBody interface{}, accessToken string, headerFunction string, path string) (res interface{}, err error) {
```
Comment (sswastioyono18) [inline core.go:238]: ini jadinya disini ya bukan di sangu dananya ya?

File: core.go (hunk 1 above)
Comment (mellyzarsmwn) [reply core.go:33]: kalo di liat sih ada hubungannya sama buat generate signature mas @sswastioyono18

File: core.go (hunk 2 above)
Comment (mellyzarsmwn) [reply core.go:238]: ini udah di sangu-dana mas

Comment (sswastioyono18) [reply core.go:238]: lah iya ga sadar wkwk

**Review by sswastioyono18 — APPROVED** [review]

**Pull request merged**